src/test/java/br/com/caelum/comidinhas/tipoCozinha/TipoCozinhaControllerTest.java

```diff
@@ -104,16 +105,19 @@ void deve_editar_retornar_um_erro_quando_o_nome_ja_existe() throws Exception{
         mockMvc.perform(put("/admin/tipos-de-cozinha/1")
                 .contentType(MediaType.APPLICATION_JSON)
                 .content(json))
-                .andExpect(status().isBadRequest());
+                .andExpect(status().isInternalServerError())
+                .andExpect(jsonPath("$.globalErrorMessages", Matchers.hasItem("O tipo de cozinha Chinesa informado nao existe")));
+        verify(tipoCozinhaRepository,never()).save(any(TipoCozinha.class));
     }
 
     @Test
-    void deve_remover_um_tipo_de_cozinhaa() throws Exception{
+    void deve_remover_um_tipo_de_cozinha() throws Exception{
         TipoCozinha italiana = new TipoCozinha(1L, "Italiana");
         when(tipoCozinhaRepository.findById(italiana.getId())).thenReturn(Optional.of(italiana));
         mockMvc.perform(delete("/admin/tipos-de-cozinha/{id}","1")
                 .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk());
+        verify(tipoCozinhaRepository,times(1)).deleteById(1L);
```

Comment:
Elias, eu lembrei que se usar só o `verify`, já checa que foi chamado apenas uma vez. Então, daria pra trocar para:

```suggestion
        verify(tipoCozinhaRepository).deleteById(1L);
```